com.tntware.TntConnect.yml

```diff
@@ -0,0 +1,197 @@
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/arialb32.exe
+        sha256: a425f0ffb6a1a5ede5b979ed6177f4f4f4fdef6ae7c302a7b7720ef332fec0a8
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/comic32.exe
+        sha256: 9c6df3feefde26d4e41d4a4fe5db2a89f9123a772594d7f59afd062625cd204e
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/courie32.exe
+        sha256: bb511d861655dde879ae552eb86b134d6fae67cb58502e6ff73ec5d9151f3384
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/georgi32.exe
+        sha256: 2c2c7dcda6606ea5cf08918fb7cd3f3359e9e84338dc690013f20cd42e930301
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/impact32.exe
+        sha256: 6061ef3b7401d9642f5dfdb5f2b376aa14663f6275e60a51207ad4facf2fccfb
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/times32.exe
+        sha256: db56595ec6ef5d3de5c24994f001f03b2a13e37cee27bc25c58f6f43e8f807ab
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/trebuc32.exe
+        sha256: 5a690d9bb8510be1b8b4fe49f1f2319651fe51bbe54775ddddd8ef0bd07fdac9
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/verdan32.exe
+        sha256: c1cb61255e363166794e47664e2f21af8e3a26cb6346eb8d2ae2fa85dd5aad96
+        dest: corefonts
+      - type: file
+        url: https://mirrors.kernel.org/gentoo/distfiles/webdin32.exe
+        sha256: 64595b5abc1080fba8610c5c34fab5863408e806aafe84653ca8575bed17d75a
+        dest: corefonts
+
+      - type: file
+        path: tntconnect.sh
+
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.tntware.TntConnect.256x256.png /app/share/icons/hicolor/256x256/apps/com.tntware.TntConnect.png
+      - install -Dm644 com.tntware.TntConnect.128x128.png /app/share/icons/hicolor/128x128/apps/com.tntware.TntConnect.png
+      - install -Dm644 com.tntware.TntConnect.64x64.png /app/share/icons/hicolor/64x64/apps/com.tntware.TntConnect.png
+      - install -Dm644 com.tntware.TntConnect.48x48.png /app/share/icons/hicolor/48x48/apps/com.tntware.TntConnect.png
+      - install -Dm644 com.tntware.TntConnect.32x32.png /app/share/icons/hicolor/32x32/apps/com.tntware.TntConnect.png
+      - install -Dm644 com.tntware.TntConnect.16x16.png /app/share/icons/hicolor/16x16/apps/com.tntware.TntConnect.png
+      - install -D --mode=644 com.tntware.TntConnect.desktop /app/share/applications/com.tntware.TntConnect.desktop
+      - install -D --mode=644 com.tntware.TntConnect.appdata.xml /app/share/metainfo/com.tntware.TntConnect.appdata.xml
+    no-make-install: true
+    sources:
+      - type: file
+        path: icons/com.tntware.TntConnect.256x256.png
```

Comment:
only 128x128 is required. Given that this icon is upscaled I think it's better to not have it.